appshell/appshell_extensions.cpp

```diff
@@ -606,6 +606,9 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
                 bool enabled = argList->GetBool(2);
                 bool checked = argList->GetBool(3);
                 error = NativeMenuModel::getInstance(getMenuParent(browser)).setMenuItemState(command, enabled, checked);
+                if (error == NO_ERROR) {
+                    error = SetMenuItemState(browser, command, enabled, checked);
```

Comment:
Actually `NativeMenuModel::getInstance(getMenuParent(browser)).setMenuItemState(command, enabled, checked);` does not work for Linux as it only sets a flag, so I have added `SetMenuState` function which is meant only for Linux and for case of Windows and MAC it is No Op